### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

**Comment by Abdur-rahmaanJ** [comment]
button_component

**Review by Ludmilla13 — DISMISSED** [review]

File: src/app/button/button.component.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent implements OnInit {
+  @Input() text: any;
+  constructor() {}
```
Comment (kherin) [inline src/app/button/button.component.ts:15]: Could you add other `(input)`attributes to customize the styling of the button:
examples:
- elevation (drop-shadow)
- background-color
- text color
- border-radius
- disabled status
-

**Review by kherin — CHANGES_REQUESTED** [review]

**Comment by Abdur-rahmaanJ** [comment]
ok will work on it

**Comment by Abdur-rahmaanJ** [comment]
btn_component_v2

**Review by kherin — APPROVED** [review]
Looks Good 👍

**Pull request merged**